modules/installation-process.adoc

```diff
@@ -72,6 +72,8 @@ If possible, use this feature to avoid having to provision and maintain the clus
 
 With installer-provisioned infrastructure clusters, {product-title} manages all aspects of the cluster, including the operating system itself. Each machine boots with a configuration that references resources hosted in the cluster that it joins. This configuration allows the cluster to manage itself as updates are applied.
 
+For some installer-provisioned infrastructure installations, for example vSphere and bare metal, the external traffic that reaches the `ingressVIPs` is not balanced between the default `IngressController` replicas. For vSphere and bare metal installer-provisioned infrastructure installations where it is required to exceed the baseline `IngressController`, or router, performance, you must configure an external load balancer. Configuring an external load balancer achieves the performance of multiple `IngressController` replicas.
```

Comment:
I don't think the information really belongs in this section. the rest of the "The installation process with installer-provisioned infrastructure" sub-section doesn't really deal with specifics.